### ISSUE TITLE
Install failing after naming website

### ISSUE THREAD

**Issue by joshRpowell** [denkGroot/Spina#102]
I'm adding Spina as the first gem in a new rails app. 
```rails v 4.2.4 / ruby 2.3.0```

running ```rails g spina:install```

i get the following trace after naming the website:
```
Running via Spring preloader in process 20063
      create  config/initializers/spina.rb
      create  config/initializers/themes/default.rb
        rake  spina:install:migrations
Copied migration 20160115135442_create_spina_tables.spina.rb from spina
        rake  db:migrate
== 20160115135442 CreateSpinaTables: migrating ================================
-- create_table("spina_accounts", {:force=>:cascade})
   -> 0.0062s
-- create_table("spina_attachment_collections", {:force=>:cascade})
   -> 0.0006s
-- create_table("spina_attachment_collections_attachments", {:force=>:cascade})
   -> 0.0007s
-- create_table("spina_attachments", {:force=>:cascade})
   -> 0.0006s
-- create_table("spina_colors", {:force=>:cascade})
   -> 0.0005s
-- create_table("spina_inquiries", {:force=>:cascade})
   -> 0.0006s
-- create_table("spina_layout_parts", {:force=>:cascade})
   -> 0.0007s
-- create_table("spina_lines", {:force=>:cascade})
   -> 0.0011s
-- create_table("spina_page_parts", {:force=>:cascade})
   -> 0.0009s
-- create_table("spina_pages", {:force=>:cascade})
   -> 0.0008s
-- create_table("spina_photo_collections", {:force=>:cascade})
   -> 0.0009s
-- create_table("spina_photo_collections_photos", {:force=>:cascade})
   -> 0.0005s
-- create_table("spina_photos", {:force=>:cascade})
   -> 0.0005s
-- create_table("spina_structure_items", {:force=>:cascade})
   -> 0.0006s
-- add_index("spina_structure_items", ["structure_id"], {:name=>"index_spina_structure_items_on_structure_id", :using=>:btree})
   -> 0.0008s
-- create_table("spina_structure_parts", {:force=>:cascade})
   -> 0.0008s
-- add_index("spina_structure_parts", ["structure_item_id"], {:name=>"index_spina_structure_parts_on_structure_item_id", :using=>:btree})
   -> 0.0006s
-- add_index("spina_structure_parts", ["structure_partable_id"], {:name=>"index_spina_structure_parts_on_structure_partable_id", :using=>:btree})
   -> 0.0010s
-- create_table("spina_structures", {:force=>:cascade})
   -> 0.0008s
-- create_table("spina_texts", {:force=>:cascade})
   -> 0.0008s
-- create_table("spina_users", {:force=>:cascade})
   -> 0.0007s
-- create_table("spina_rewrite_rules", {:force=>:cascade})
   -> 0.0010s
== 20160115135442 CreateSpinaTables: migrated (0.0233s) =======================

      create  app/views/layouts/default/application.html.erb
      create  app/views/default/pages/homepage.html.erb
      create  app/views/default/pages/show.html.erb
What would you like to name your website? spina
/Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activerecord-4.2.4/lib/active_record/attribute_assignment.rb:59:in `rescue in _assign_attribute': unknown attribute 'theme' for Spina::Account. (ActiveRecord::UnknownAttributeError)
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activerecord-4.2.4/lib/active_record/attribute_assignment.rb:54:in `_assign_attribute'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activerecord-4.2.4/lib/active_record/attribute_assignment.rb:41:in `block in assign_attributes'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activerecord-4.2.4/lib/active_record/attribute_assignment.rb:35:in `each'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activerecord-4.2.4/lib/active_record/attribute_assignment.rb:35:in `assign_attributes'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activerecord-4.2.4/lib/active_record/core.rb:564:in `init_attributes'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activerecord-4.2.4/lib/active_record/core.rb:281:in `initialize'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activerecord-4.2.4/lib/active_record/inheritance.rb:61:in `new'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activerecord-4.2.4/lib/active_record/inheritance.rb:61:in `new'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activerecord-4.2.4/lib/active_record/persistence.rb:33:in `create'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/spina-0.7.0/lib/generators/spina/install_generator.rb:29:in `create_account'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `block in invoke_all'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `each'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `map'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `invoke_all'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/thor-0.19.1/lib/thor/group.rb:232:in `dispatch'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/railties-4.2.4/lib/rails/generators.rb:157:in `invoke'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/railties-4.2.4/lib/rails/commands/generate.rb:13:in `<top (required)>'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:274:in `require'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:274:in `block in require'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:274:in `require'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/railties-4.2.4/lib/rails/commands/commands_tasks.rb:123:in `require_command!'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/railties-4.2.4/lib/rails/commands/commands_tasks.rb:130:in `generate_or_destroy'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/railties-4.2.4/lib/rails/commands/commands_tasks.rb:50:in `generate'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/railties-4.2.4/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/railties-4.2.4/lib/rails/commands.rb:17:in `<top (required)>'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:274:in `require'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:274:in `block in require'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:274:in `require'
	from /Users/jpowell/Dropbox/Demo/spina2/bin/rails:9:in `<top (required)>'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:268:in `load'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:268:in `block in load'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /Users/jpowell/.rvm/gems/ruby-2.3.0/gems/activesupport-4.2.4/lib/active_support/dependencies.rb:268:in `load'
	from /Users/jpowell/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/jpowell/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from -e:1:in `<main>'
```

**Comment by abukhajilnour** [comment]
Did you find the solution?? I have the same error

**Comment by Bramjetten** [comment]
Could you perhaps try using the version from the master branch and see if that one works?

**Comment by abukhajilnour** [comment]
/Users/nour/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/activerecord-4.1.10/lib/active_record/attribute_assignment.rb:50:in `rescue in _assign_attribute': unknown attribute: name (ActiveRecord::UnknownAttributeError)

Could you perhaps try using the version from the master branch and see if that one works
How to do that ? can you explain more

**Comment by Bramjetten** [comment]
It's working for me on a brand new app with Ruby 2.2.3 and Rails 4.2.5.

**Comment by abukhajilnour** [comment]
I work on this gem 'rails', '4.1.10'

**Comment by abukhajilnour** [comment]
Please help me on this I have to do CMS to my ROR app

**Comment by Bramjetten** [comment]
Does it work in a clean project?

**Comment by abukhajilnour** [comment]
I didnt try

**Comment by abukhajilnour** [comment]
It work in another project but I need it for this project

**Comment by xijo** [comment]
I'm getting `.gem/ruby/2.3.0/gems/activerecord-4.2.5/lib/active_record/attribute_assignment.rb:59:in `rescue in _assign_attribute': unknown attribute 'theme' for Spina::Account. (ActiveRecord::UnknownAttributeError)` in an existing ruby-2.3.0/rails-4.2.5 application as well. Having a quick look at Spina::Account it looks like there is no such attribute accessor or db column as `theme` - any ideas how to debug it further?

**Comment by Bramjetten** [comment]
It's defined dynamically in self.serialized_attr_accessor.

**Comment by xijo** [comment]
@Bramjetten If I use the github version everything's fine. Looks like you already fixed it and just need to do a new release.

**Comment by xijo** [comment]
But there is another problem: `private method `theme' called for #<Spina::Account:0x007fef21728310>
`

I guess self.serialized_attr_accessor mustn't be private in order to work correctly.

**Comment by Bramjetten** [comment]
You're right. Fixed in newest release: 0.7.3.

**Comment by xijo** [comment]
@Bramjetten very nice, thanks for your work
